Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.BeEquivalentTo.cs

```diff
@@ -158,6 +158,50 @@ public void Default_immutable_lists_should_be_equivalent()
             // Act / Assert
             collection.Should().BeEquivalentTo(collection1);
         }
+
+        [Fact]
+        public void Succeed_for_collection_with_case_different_strings_when_IgnoringCase()
```

Comment:
```suggestion
        public void Can_ignore_casing_while_comparing_collections_of_strings()
```

We definitely _do not_ want to include names of code elements in test names. We also prefer to avoid terms like _when_ and _should_. Same for the other tests.

Reply:
Done!